Failed: JDK9 ClassNotFoundException: javax.xml.bind.annotation.XmlRootElement

```
Uncaught error from thread [NonSharedCamelSystem-akka.actor.default-dispatcher-8]: javax/xml/bind/annotation/XmlRootElement, shutting down JVM since 'akka.jvm-exit-on-fatal-error' is enabled for ActorSystem[NonSharedCamelSystem]
java.lang.NoClassDefFoundError: javax/xml/bind/annotation/XmlRootElement
	at org.apache.camel.model.OptionalIdentifiedDefinition.getShortName(OptionalIdentifiedDefinition.java:79)
	at org.apache.camel.impl.DefaultNodeIdFactory.createId(DefaultNodeIdFactory.java:36)
	at org.apache.camel.model.OptionalIdentifiedDefinition.idOrCreate(OptionalIdentifiedDefinition.java:158)
	at org.apache.camel.model.ProcessorDefinition.createOutputsProcessorImpl(ProcessorDefinition.java:455)
	at org.apache.camel.model.ProcessorDefinition.createOutputsProcessor(ProcessorDefinition.java:418)
	at org.apache.camel.model.ProcessorDefinition.createOutputsProcessor(ProcessorDefinition.java:165)
	at org.apache.camel.model.OnExceptionDefinition.addRoutes(OnExceptionDefinition.java:198)
	at org.apache.camel.model.RouteDefinition.addRoutes(RouteDefinition.java:1069)
	at org.apache.camel.model.RouteDefinition.addRoutes(RouteDefinition.java:196)
	at org.apache.camel.impl.DefaultCamelContext.startRoute(DefaultCamelContext.java:974)
	at org.apache.camel.impl.DefaultCamelContext.startRouteDefinitions(DefaultCamelContext.java:3301)
	at org.apache.camel.impl.DefaultCamelContext.addRouteDefinitions(DefaultCamelContext.java:908)
	at org.apache.camel.builder.RouteBuilder.populateRoutes(RouteBuilder.java:418)
	at org.apache.camel.builder.RouteBuilder.addRoutesToCamelContext(RouteBuilder.java:331)
	at org.apache.camel.impl.DefaultCamelContext$1.call(DefaultCamelContext.java:835)
	at org.apache.camel.impl.DefaultCamelContext$1.call(DefaultCamelContext.java:832)
	at org.apache.camel.impl.DefaultCamelContext.doWithDefinedClassLoader(DefaultCamelContext.java:2873)
	at org.apache.camel.impl.DefaultCamelContext.addRoutes(DefaultCamelContext.java:832)
	at akka.camel.internal.ConsumerRegistrar$$anonfun$receive$4.applyOrElse(CamelSupervisor.scala:201)
	at akka.actor.Actor.aroundReceive(Actor.scala:517)
	at akka.actor.Actor.aroundReceive$(Actor.scala:515)
	at akka.camel.internal.ConsumerRegistrar.aroundReceive(CamelSupervisor.scala:196)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:588)
	at akka.actor.ActorCell.invoke(ActorCell.scala:557)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:258)
	at akka.dispatch.Mailbox.run(Mailbox.scala:225)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:235)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.annotation.XmlRootElement
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:582)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:190)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:499)
	... 31 more
```

--- a/build.sbt
+++ b/build.sbt
@@ -190,7 +190,8 @@ lazy val distributedData = akkaModule("akka-distributed-data")
 
 lazy val docs = akkaModule("akka-docs")
   .dependsOn(
-    actor, cluster, clusterMetrics, slf4j, agent, camel, osgi, persistenceTck, persistenceQuery, distributedData, stream,
+    actor, cluster, clusterMetrics, slf4j, agent, osgi, persistenceTck, persistenceQuery, distributedData, stream,
+    camel % "compile->compile;test->test",
     clusterTools % "compile->compile;test->test",
     clusterSharding % "compile->compile;test->test",
     testkit % "compile->compile;test->test",

